res/controllers/Traktor-Kontrol-S4-MK2-hid-scripts.js

```diff
@@ -679,7 +679,7 @@ TraktorS4MK2.lightDeck = function(group) {
 };
 
 TraktorS4MK2.pointlessLightShow = function() {
-    var packets = [Object(), Object(), Object()];
+    const packets = [[], [], []];
```

Comment:
eslint insisted on `const` here.
So const is only the number of items in the array? Not the size/content of the items?

Reply:
yup. `const` in JS only applies to the binding. Meaning a variable assigned with `const` can not be reassigned. For primitive values (numbers, bools, etc) that means they're effectively const. For complex types (Objects, arrays, strings, etc) you can never reassign the value, but you can mutate it. So yeah, `const` is correct in this case.

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/const

Reply:
Thanks!